MvvmCross/MvvmCross.csproj

```diff
@@ -13,15 +13,23 @@
     <None Include="Resources\*.cs" />
     <Compile Remove="Resources\*.cs" />
   </ItemGroup>
-  
+
   <ItemGroup>
-       <PackageReference Include="Microsoft.CSharp" Version="4.4.1" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.4.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.1'">
+    <Compile Include="Platform\Netstandard1.1\**\*.cs" />
   </ItemGroup>
   
+  <ItemGroup Condition=" '$(TargetFramework)' != 'netstandard1.1'">
+    <Compile Include="Platform\ReflectionExtensions.cs" />
```

Comment:
This should probably go somewhere else. I think we should add a `Netstandard2.0` folder, and have a shared `Netstandard` folder.